### ISSUE TITLE
[Bug]: sp-grid select checkbox pointer target too small

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#2527]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-grid

### Expected behavior

On the grid storybook demos the target for the select is too small. The entire card should allow selection.

(note this may just be a feature of the stories but they should be more realistic for normal usage)

### Actual behavior

Only the checkbox can be clicked to select. This is too small of a target

### What browsers are you seeing the problem in?

Firefox, Chrome, Safari, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/grid--default
2. Click on a card
3. note that you have to click in the checkbox itself to select

**Comment by najikahalsema** [comment]
SWC-204

**Comment by castastrophe** [comment]
Hi James! We've got this tracking in our accessibility epic, please follow along there for updates!
